docs/plone-deployment/new-project.md

```diff
@@ -23,9 +26,9 @@ You can accept the default values in square brackets (`[default-option]`) by hit
 
 For this training, we recommend you to provide:
 
-* `project_title`: **Plone Conference**
-* `github_organization`: Your GitHub username
-* `container_registry`: **2**
+- `project_title`: **Plone Conference**
+- `github_organization`: Your GitHub username
+- `container_registry`: **2**
```

Comment:
I am OK with this automatic editor change because it requires one fewer keystroke. :grin:

Reply:
I learned my lesson with autoformatting tools 🙃